app/routes/content.js

```diff
@@ -104,7 +104,8 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
                 ['og:description', node.get('description')],
                 ['og:site_name', this.get('theme.provider.name')],
                 ['og:type', 'article'],
-                ['article:published_time', new Date(preprint.get('dateCreated') || null).toISOString()]
+                ['article:published_time', new Date(preprint.get('datePublished')).toISOString()],
```

Comment:
So I need to restore || null, and the Date(...) needs to stay because we don't want to do `.toISOString` of null.